modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,244 @@
+|===
+
+==== Associated Operations
+
+* <<Route>> 
+* <<Serialize>> 
+* <<SerializeBatch>> 
+
+==== Associated Sources
+
+* <<DataFetcher>> 
+* <<DataLoader>> 
+
+== Operations
+
+=== Route
+
+`<graphql-router:route>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | *x*
+| Content | Any |  |  #[payload] | 
+| Output MIME Type | String |  The MIME type of the payload that this operation outputs. |  | 
+| Output Encoding | String |  The encoding of the payload that this operation outputs. |  | 
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Streaming Strategy  a| 
+* <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configure if repeatable streams should be used and their behavior. |  | 
+| Target Variable | String |  The name of a variable on which the operation output is placed. |  | 
+| Target Value | String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
+
+=== Serialize
+
+`<graphql-router:serialize>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Inbound Content | Any |  |  #[payload] | 
+| Config Ref | ConfigurationProvider |  The name of the configuration you use to execute this component. |  | *x*
+| Object Type Name | String |  The object type name |  | *x*
+| Field Name | String |  The field name |  | 
+| Target Variable | String |  The name of a variable on which the operation output is placed. |  | 
+| Target Value | String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
+
+=== Serialize Batch
+
+`<graphql-router:serialize-batch>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Object Type Name | String | The object type name |  | *x*
+| Inbound Content | Any |  |  #[payload] | 
+| Config Ref | ConfigurationProvider |  The name of the configuration you use to execute this component. |  | *x*
+| Target Variable a| String |  The name of a variable on which the operation output is placed. |  | 
+| Target Value | String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
```

Comment:
Should this link to the "=== Config" title?
If so, we link to content with an xref: xref:filename.adoc#articletitle[title or phrase to explain title action]

Reply:
Yes

Reply:
OK! Thanks!